api/core/routes.py

```diff
@@ -92,30 +83,29 @@ def get_ephemeris_by_name():
         abort(400) 
 
     #Cast the latitude, longitude, and jd to floats (request parses as a string)
-    lat = float(latitude)
-    lon = float(longitude)
-    ele = float(elevation)
-    
+    try:
+        location = EarthLocation(lat=float(latitude)*u.deg, lon=float(longitude)*u.deg, height=float(elevation)*u.m)
+    except:
+        abort(500, "Error: Invalid location parameters")
+
     # Converting string to list
-    jul = str(julian_date).replace("%20", ' ').strip('][').split(', ')
-   
-    # Converting list elements to float
-    jd = [float(i) for i in jul]
+    try:
+        jd = Time(julian_date, format='jd', scale='ut1')
+    except:
+        abort(500, "Error: Invalid Julian Date")
```

Comment:
Thanks for switching to astropy time objects! I like the try/except blocks throughout, but can you please specify the exception instead of catching any/all exceptions? In this case I think it would be `except ValueError:`. I'm not familiar with the flask "abort" but it seems like without a valid jd things can't continue and therefore reasonable.